File: launcher-joke/src/main/java/mapper/InheritConfiguration.java
package mapper;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * 复用另一映射方法配置的简化注解。
 */
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.CLASS)
public @interface InheritConfiguration {

    /**
     * 需要继承配置的方法名。
     */
    String name();
}

File: launcher-joke/src/main/java/mapper/InheritConfiguration.kt
package mapper

/**
 * 复用另一映射方法配置的简化注解。
 */
@Target(AnnotationTarget.FUNCTION)
@Retention(AnnotationRetention.BINARY)
annotation class InheritConfiguration(
    /**
     * 需要继承配置的方法名。
     */
    val name: String
)
